pkgs/development/python-modules/tempest/default.nix

```diff
@@ -36,6 +36,14 @@ buildPythonApplication rec {
     sha256 = "0521d3042360c0fb469b16f99174a9abddbae8a2d2a81268cfc664f1ccfdd0f9";
   };
 
+  patches = [
+    # remove need for unittest2
+    (fetchpatch {
+      url = "https://github.com/openstack/tempest/pull/32/commits/cd3745c27b7d8fcdaffc72b965a3d803d9ee12c2.patch";
```

Comment:
```suggestion
      url = "https://github.com/openstack/tempest/commits/cd3745c27b7d8fcdaffc72b965a3d803d9ee12c2.patch";
```

Reply:
Openstack does not work with pull requests. You need to go through their lovely Gerrit and unless you know someone no one probably looks at the changes.

Reply:
this is annoying, why don't they say it's a read-only mirror then?